README.md

```diff
@@ -2,13 +2,30 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+# TAilor
+
+TAilor is **desktop app for managing contacts, optimized for use via a Command Line Interface (CLI)** while still
+having the benefits of a Graphical User Interface (GUI). If you are a Computer Science Teaching Assistant who can type 
+fast, TAilor can get your contact management tasks done faster than traditional GUI apps.
+<br>
+
+## Usages
+TAilor can (as of v1.2):
+* add, delete, edit and view student's contact details
+* store the contact details of your students
+* contact students in bulk or individually
+
+## Contact
+To reach any of our developers, you can find our contact details in AboutUs.md, which can be found in the docs folder.
```

Comment:
hello! would it be nice to just add our emails here too? Just to reduce the amount of navigation someone needs to engage in to simply contact us?